Apsitvarkom.Models/Mapping/CleaningEventProfile.cs

```diff
@@ -17,7 +17,8 @@ public CleaningEventProfile()
 
     private void MapRequests()
     {
-
+        CreateMap<ObjectIdentifyRequest, CleaningEvent>(MemberList.None).
```

Comment:
Kind of a styling related nitpick, but looking at where to place the dot (before or after the line break) or similar questions (for example 
```
a
+ b
``` 
or 
```
a + 
b
```
I usually refer to [this](https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break) article. That's the usual practices used at Google (of course in a work scenario, you would instead keep up with what is around you or a preferred coding style). 
That article suggests having the dot in the new line [4.5.1.1], so maybe let's just sliiightly adjust this?

Reply:
Sure